core/controllers/base.py

```diff
@@ -504,7 +547,15 @@ def init_csrf_secret(cls):
 
     @classmethod
     def _create_token(cls, user_id, issued_on):
-        """Creates a digest (string representation) of a token."""
+        """Creates a new CSRF token.
+
+        Args:
+            user_id: str. The user_id for whom token is generated
```

Comment:
@sean 
@seanlip This is [the doc](https://webapp2.readthedocs.io/en/latest/api/webapp2.html#webapp2.WSGIApplication.handle_exception) for `handle_exception` I have seen 
```
Parameters:	

request – A Request instance.
response – A Response instance.
e – The uncaught exception.

Returns:	
The returned value from the error handler.
```
Is it not the correct one